streaming/base/dataset.py

```diff
@@ -319,13 +320,13 @@ def __init__(self,
                  cache_limit: Optional[Union[int, str]] = None,
                  sampling_method: str = 'balanced',
                  sampling_granularity: int = 1,
-                 partition_algo: str = 'orig',
+                 partition_algo: str = 'relaxed',
```

Comment:
Do we have enough evidence to make a `relaxed` partition algo as a default algorithm?

Reply:
It uses the `orig` partition algo in all of the cases where it was applicable, and only implements new logic when NCN and PN divisibility constraints are not satisfied, so imo it should be fine!